packages/roosterjs-editor-types/lib/event/ShadowEditEvent.ts

```diff
@@ -14,7 +14,7 @@ export interface EnterShadowEditEvent extends BasePluginEvent<PluginEventType.En
     /**
      * The selection path of original editor content
      */
-    selectionPath: SelectionPath;
+    selectionPath: SelectionPath[];
```

Comment:
Do not change existing var definition, you can add additional property TableSelection